src/tools/share/python/doc_bunchInit.md

```diff
@@ -0,0 +1,219 @@
+```python
```

Comment:
this file (as `.rst` file) can go in the `docs/source/usage/workflows/` section.